src/display/editor/tools.js

```diff
@@ -957,9 +957,11 @@ class AnnotationEditorUIManager {
       "highlightColors",
       this.#highlightColors
         ? new Map(
-            this.#highlightColors
-              .split(",")
-              .map(pair => pair.split("=").map(x => x.trim()))
+            this.#highlightColors.split(",").map(pair => {
+              pair = pair.split("=").map(x => x.trim());
+              pair[1] = pair[1].toUpperCase();
```

Comment:
What's the purpose of upper-casing the color strings in this patch exactly? I'm not really clear on why that is necessary to improve accessibility.

Reply:
I noticed that the set color in color picker was lowercase and it was making this test:
https://github.com/mozilla/pdf.js/blob/ac399e77e8dd7961a1a8d54eeaed593463c2e02a/src/display/editor/color_picker.js#L139
failing, so no color was aria-selected.
I could have fixed the color case when creating it, but then I thought that since the color can be changed in the pref/options, it was better to always make a case-independent test.
Anyway, fixing the aria-selected value is an a11y improvement.